ckanext/benap/i18n/ckanext-benap.pot

```diff
@@ -33,6 +33,14 @@ msgstr ""
 msgid "URL {url} has to start with https://"
 msgstr ""
 
+#: ckanext/benap/validators.py:92
+msgid "Modification must done by a system administrator"
```

Comment:
Vreemde zin, misschien beter: "Modification must be done by a system administrator." (puntje niet vergeten achter de zin)

Reply:
Staat als info onder een form element, lijkt me overbodig